ambari-server/src/main/resources/common-services/AMBARI_LOGSEARCH/3.0.0/package/alerts/alert_logfeeder.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
```

Comment:
https://github.com/apache/ambari/blob/e8104f0a11a01ba02951500eb19cb9b03a8bfa18/ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/services/HIVE/package/scripts/hcat.py#L1     As the Ambari trunk is currently using Python 3, you can refer to these services and uniformly set the Python interpreter to Python 3.